### PR TITLE
Using project reference

### PR DESCRIPTION
**Pull request by lock9**
Referencing project instead of nuget

### CODE REVIEW

**Comment by Ashuaidehao** [comment]
Do you mean to add neo as a submodule? Only reference neo with your local machine's relative path won't work in others machine.

**Comment by shargon** [comment]
I prefer with submodules too

File: neo-cli/neo-cli.csproj [hunk 1]
```diff
@@ -28,8 +28,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="3.0.0-CI00130" />
     <PackageReference Include="System.ServiceProcess.ServiceController" Version="4.5.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\neo\neo\neo.csproj" />
```
Comment (vncoelho) [inline neo-cli/neo-cli.csproj]: This works, @lock9, since they are under the same `github` account?

**Comment by shargon** [comment]
it works but is a little bit weird, i think that submodules will be better here

**Pull request closed without merging**